lib/compat/wordpress-6.1/script-loader.php

```diff
@@ -73,6 +73,23 @@ function gutenberg_enqueue_global_styles() {
 	gutenberg_add_global_styles_for_blocks();
 }
 
+/**
+ * Enqueues the global styles custom css.
+ *
+ * @since 6.2.0
+ */
+function gutenberg_enqueue_global_styles_custom_css() {
+	$custom_css     = get_global_styles_custom_css();
+	$is_block_theme = wp_is_block_theme();
+	if ( $custom_css && $is_block_theme ) {
+		?>
```

Comment:
Outputting the style element here, instead of using `wp_enqueue_style` was the only way I could get this to load in the header after the customizer custom CSS - there may be a better approach that I am overlooking